scipy/stats/tests/test_distributions.py

```diff
@@ -1781,6 +1781,16 @@ def test_entropy_2d_zero(self):
         assert_array_almost_equal(stats.entropy(pk, qk),
                                   [0.17403988, 0.18609809])
 
+    def test_entropy_base_2d_nondefault_axis(self):
+        pk = [[0.1, 0.2], [0.6, 0.3], [0.3, 0.5]]
+        assert_array_almost_equal(stats.entropy(pk, axis=1),
+                                  [0.63651417, 0.63651417, 0.66156324])
+
+    def test_entropy_2d_nondefault_axis(self):
+        pk = [[0.1, 0.2], [0.6, 0.3], [0.3, 0.5]]
+        qk = [[0.2, 0.1], [0.3, 0.6], [0.5, 0.3]]
+        assert_array_almost_equal(stats.entropy(pk, qk, axis=1),
+                                  [0.231049, 0.231049, 0.127706])
```

Comment:
It would be nice to have a test that `stats.entropy(pk.T, qk.T)` is equivalent to  `stats.entropy(pk, qk, axis=1)`. (And similar for `qk` not given)

Reply:
I assume you meant axis=0, I added a test for this

Reply:
>I assume you meant axis=0

No, I mean that computing over `axis=1` should be equivalent to `axis=0` of a transposed array. Although I missed out the final transpose; it should be `statis.entropy(pk.T, qk.T, axis=0).T` compared to `stats.entropy(pk, qk, axis=1)`.

Reply:
makes sense!